### PR TITLE
Fixed typo in README and added instructions on how to virtualenv

### PR DESCRIPTION
**Pull request by dnshio**
This looks like a really great open source project. I'd like to contribute to its development so I started reading your docs. I noticed a couple of small issues there so here's a PR to fix them. 

With regards to instructions on `virtualenv`, I'm open to feedback on whether I should include more detail. Also, my improvement only recommends python's `venv` module rather than the independent [virtualenv](https://virtualenv.pypa.io/en/stable/) package. I believe in recommending a single tool and and python's `venv` module is best because it doesn't required additional installation. 

Let me know how I could improve this PR.

### CODE REVIEW

**Comment by NiallRees** [comment]
> This looks like a really great open source project. I'd like to contribute to its development so I started reading your docs. I noticed a couple of small issues there so here's a PR to fix them.
> 
> With regards to instructions on `virtualenv`, I'm open to feedback on whether I should include more detail. Also, my improvement only recommends python's `venv` module rather than the independent [virtualenv](https://virtualenv.pypa.io/en/stable/) package. I believe in recommending a single tool and and python's `venv` module is best because it doesn't required additional installation.
> 
> Let me know how I could improve this PR.

Yes Dinesh! I'm personally a big pipenv fan, have you used it and do you think it could be a good choice here rather than venv?

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 [![CircleCI](https://img.shields.io/circleci/build/gh/sqlfluff/sqlfluff/master?style=flat-square&logo=CircleCI)](https://circleci.com/gh/sqlfluff/sqlfluff/tree/master)
 [![ReadTheDocs](https://img.shields.io/readthedocs/sqlfluff?style=flat-square&logo=Read%20the%20Docs)](https://sqlfluff.readthedocs.io)
 
-Bored of not having a good SQL linter that works with whichever dialiect you're
+Bored of not having a good SQL linter that works with whichever dialect you're
```
Comment (pwildenhain) [inline README.md:16]: I _just_ noticed this as well the other day, thanks for fixing!

**Review by NiallRees — APPROVED** [review]

**Comment by pwildenhain** [comment]
>Yes Dinesh! I'm personally a big pipenv fan, have you used it and do you think it could be a good choice here rather than venv? 

Personally, my vote is for `venv` 🤚

**Comment by NiallRees** [comment]
Out of interest are you able to merge this now I've approved it @dnshio ? Or do I need to

**Comment by dnshio** [comment]
> > Yes Dinesh! I'm personally a big pipenv fan, have you used it and do you think it could be a good choice here rather than venv?
> 
> Personally, my vote is for `venv` 🤚 though if we start to run into tricky dependency issues, I could see opting for `pipenv` instead. [`poetry`](https://python-poetry.org/) is also worth considering if you're going to consider `pipenv` since it's built specifically for packages.

Have used `pipenv` for a couple of projects and I ended up ditching it in the end because it ended up being more hassle than what it was worth. Maybe I didn't see the true power of it. But I grew to appreciate the simplicity of having either a `setup.py` or a `requirements.txt` with pinned (to point versions) and let the client decide what virtual environment they want to use.

File: README.md (hunk 1 above)
Comment (dnshio) [reply README.md:16]: 😄  I thought this was intentionally left there for new comers!

**Comment by dnshio** [comment]
> Out of interest are you able to merge this now I've approved it @dnshio ? Or do I need to

I do not have permission to merge. Would be great if you could :)

**Pull request merged**